ci-operator/step-registry/cri-o/cgroupv2/OWNERS

```diff
@@ -1,5 +1,14 @@
 approvers:
-- wgahnagl
-- rphillips
```

Comment:
can we add @rphillips to the list of OWNERS as well?

Reply:
Does it make sense to add active cri-o developers to this list rather than syncing the cri-o OWNERS list?

Reply:
> can we add @rphillips to the list of OWNERS as well?

Yes, sure

> Does it make sense to add active cri-o developers to this list here rather than syncing the cri-o OWNERS list?
I think we should keep the CRI-O OWNERS in sync with related repositories. We also can't add non Red Hat folks, but that's given by nature.